### PR TITLE
Stage

### PR DESCRIPTION
**Pull request by myanzik**

### CODE REVIEW

File: src/containers/our-ecosystem/index.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import clsx from "clsx";
 import PropTypes from "prop-types";
 // import Button from "@ui/button";
 import ReadMoreBtn from "@components/read-more-button";
-import Image from "next/image";
+import { Image } from "react-bootstrap";
```
Comment (ManiByanjankar) [inline src/containers/our-ecosystem/index.jsx:6]: why importing 'Image' from react-bootstrap? it should be from "next/image"

File: src/containers/our-ecosystem/index.jsx [hunk 2]
```diff
@@ -30,57 +30,56 @@ const OurEcosystem = ({ space, className }) => {
...
                         <div className="rn-service-one-our-ecosystem color-shape-7">
                             <div className="inner">
-                                <div className="thumbnail">
+                                <div className="thumbnail text-center">
                                     <Image
                                         src="https://rumsan.nyc3.cdn.digitaloceanspaces.com/esatya/cva-ecosystem-banner.jpg"
                                         alt="cva-ecosystem-banner"
-                                        width={489}
-                                        height={366}
+                                        style={{
```
Comment (ManiByanjankar) [inline src/containers/our-ecosystem/index.jsx:37]: Why did you change this width and height? Lets use 'Image' from "next/image"

File: src/assets/css/bootstrap.min.css [hunk 3]
```diff
@@ -11280,3 +11283,19 @@ fieldset:disabled .btn {
         display: none !important;
     }
 }
+
+/*  */
+
+.custom-link {
```
Comment (myanzik) [inline src/assets/css/bootstrap.min.css:11289]: Why are you making changes in bootstrap.min.css. This file should never be changed?

File: src/data/fin-dig-liter.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+[
+    {
+        "image": "/images/pic/fin&liter01.jpg.jpg"
```
[the remark this answers is not in the document]
Comment (karun-rumsan) [reply src/data/fin-dig-liter.json:3]: i was not guided to put the the image from manager.rumsan.com.

File: src/assets/css/bootstrap.min.css (hunk 3 above)
[the remark this answers is not in the document]
Comment (karun-rumsan) [reply src/assets/css/bootstrap.min.css:11289]: ok

Comment (karun-rumsan) [reply src/assets/css/bootstrap.min.css:11289]: done

File: src/containers/our-ecosystem/index.jsx [hunk 5]
```diff
@@ -90,31 +89,29 @@ const OurEcosystem = ({ space, className }) => {
...
                             <div className="inner">
-                                <div className="thumbnail">
+                                <div className="thumbnail text-center object-fit-cover">
                                     <Image
-                                        src="https://rumsan.nyc3.cdn.digitaloceanspaces.com/esatya/anticipatory-actions-banner.jpg"
+                                        src="/images/pic/fin04.jpg"
                                         alt="anticipatory-actions-banner"
-                                        width={489}
-                                        height={366}
+                                        className="img-fluid"
```
[the remark this answers is not in the document]
Comment (karun-rumsan) [reply src/containers/our-ecosystem/index.jsx:96]: i was not guided to put the the image from manager.rumsan.com.

File: src/containers/our-ecosystem/index.jsx [hunk 6]
```diff
@@ -30,57 +30,56 @@ const OurEcosystem = ({ space, className }) => {
...
                         <div className="rn-service-one-our-ecosystem color-shape-7">
                             <div className="inner">
-                                <div className="thumbnail">
+                                <div className="thumbnail text-center">
                                     <Image
                                         src="https://rumsan.nyc3.cdn.digitaloceanspaces.com/esatya/rahat-beema-banner.jpg"
                                         alt="rahat-beema-banner"
-                                        width={489}
-                                        height={366}
+                                        style={{
```
[the remark this answers is not in the document]
Comment (karun-rumsan) [reply src/containers/our-ecosystem/index.jsx:67]: i was not guided to put the the image from manager.rumsan.com.

File: src/containers/our-ecosystem/index.jsx (hunk 2 above)
Comment (karun-rumsan) [reply src/containers/our-ecosystem/index.jsx:37]: I was not guided  to use manager.rumsan.com  so i kept the image in public  and need to use the react/bootstrap instead of next/image as the image were not  rendering the  as  the width and height  mentioned

File: src/containers/our-ecosystem/index.jsx (hunk 1 above)
Comment (karun-rumsan) [reply src/containers/our-ecosystem/index.jsx:6]: I was not guided to use manager.rumsan.com so i kept the image in public and need to use the react/bootstrap instead of next/image as the image were not rendering the as the width and height mentioned

File: src/assets/css/bootstrap.min.css (hunk 3 above)
Comment (myanzik) [reply src/assets/css/bootstrap.min.css:11289]: @karun-rumsan OK. Make the changes and send PR to stage.

File: src/containers/our-ecosystem/index.jsx (hunk 2 above)
Comment (myanzik) [reply src/containers/our-ecosystem/index.jsx:37]: @karun-rumsan Now that you know. Get access to manager.rumsan.com from mani and make the required changes.

Lets try to maintain uniformity in code.
